spec/lib/open_food_network/order_cycle_management_report_spec.rb

```diff
@@ -26,6 +26,14 @@ module OpenFoodNetwork
           expect(subject.orders).to eq([o2])
         end
 
+        it 'orders them by id' do
+          result = instance_double(ActiveRecord::Relation)
+          allow_any_instance_of(Ransack::Search).to receive(:result).and_return(result)
+          expect(result).to receive(:order).with(:id) { Spree::Order.none }
```

Comment:
I think that this spec is too specific. I would create three orders with their ids in irregular order and then test that they are returned in the right order.

This spec would incorrectly pass if the code called `result.order(:completed_at).order(:id)`.

Reply:
Yes, it's very brittle so I might as well remove it. We can't fake ids to be in irregular order AFAIK. It's the db who assigns them and they'll always follow the order in which they were created.

Reply:
We could as well orde by completed_at which might be better in terms of UX

Reply:
> We can't fake ids to be in irregular order AFAIK.

We can! As long as there's no conflict, of course:

```
a = AdjustmentMetadata.order(:id).last
=> #<AdjustmentMetadata id: 1737, adjustment_id: 1739, enterprise_id: 3, fee_name: "markup", fee_type: "sales", enterprise_role: "coordinator">

b = AdjustmentMetadata.create(a.attributes.merge(id: a.id + 5))
=> #<AdjustmentMetadata id: 1742, adjustment_id: 1739, enterprise_id: 3, fee_name: "markup", fee_type: "sales", enterprise_role: "coordinator">
```

But I agree that ordering by another column is a much better idea. `created_at` is closest to `id` but maybe Rachel knows which default order the most sense.

Reply:
Solved in the last commit.